advanced_functionality/huggingface_byo_scripts_and_data/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2886 @@
+{
```

Comment:
<p>Please clear all notebook outputs. If you want to include any additional figures, please use markdown or separate JPG images.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Noted